### ISSUE TITLE
Cannot setup InfluxDB export (ModuleNotFound) docker

### ISSUE THREAD

**Issue by Trolann** [nicolargo/glances#2147]
Before filling this issue, please read the manual (https://glances.readthedocs.io/en/latest/) and search if the bug do not already exists in the database (https://github.com/nicolargo/glances/issues).

For any questions concerning installation or use, please open a discussion (https://github.com/nicolargo/glances/discussions), not an issue.

#### Description

Setting up a new instance of glances yields the following error with the given configs:
```
ModuleNotFoundError: No module named 'influxdb_client'
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/runpy.py", line 196, in _run_module_as_main
  File "/usr/local/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
    glances.main()
  File "/usr/local/lib/python3.10/site-packages/glances/__main__.py", line 19, in <module>
  File "/usr/local/lib/python3.10/site-packages/glances/__init__.py", line 179, in main
    start(config=core.get_config(), args=core.get_args())
  File "/usr/local/lib/python3.10/site-packages/glances/__init__.py", line 108, in start
    self.stats = GlancesStats(config=config, args=args)
    from influxdb_client import InfluxDBClient, WriteOptions
    export_module = __import__(self._exports_all[export_name])
  File "/usr/local/lib/python3.10/site-packages/glances/exports/glances_influxdb2.py", line 18, in <module>
  File "/usr/local/lib/python3.10/site-packages/glances/stats.py", line 162, in load_exports
    self.load_exports(args=args)
  File "/usr/local/lib/python3.10/site-packages/glances/stats.py", line 90, in load_modules
    self.load_modules(self.args)
  File "/usr/local/lib/python3.10/site-packages/glances/stats.py", line 39, in __init__
  File "/usr/local/lib/python3.10/site-packages/glances/standalone.py", line 45, in __init__
    mode = GlancesMode(config=config, args=args)
```

glances.conf
``` [ influxdb ]
# Configuration for the --export influxdb option
# https://influxdb.com/
host=192.168.86.6
port=8086
user=glances
password=1234
db=glances
prefix=fb
#tags=foo:bar,spam:eggs
```

Docker Compose snippet:
```
  glances:
    container_name: glances
    restart: always
    ports:
        - '61208-61209:61208-61209'
    environment:
        - GLANCES_OPT=-w
    volumes:
        - '/var/run/docker.sock:/var/run/docker.sock:ro'
		- '/home/fb/docker/glances/glances.conf:/glances/conf/glances.conf
    pid: host
    image: docker.io/nicolargo/glances
```

#### Versions

* Glances & psutil (glances -V): :latest tag
* Operating System (lsb_release -a): :latest tag
* How do you install Glances (Pypi package, script, package manager, source): docker

#### Logs

Given above, fails on start

**Comment by Trolann** [comment]
I did see influxdb is being deprecated in the [docker-compose glances.conf](https://github.com/nicolargo/glances/blob/develop/docker-compose/glances.conf), but also not sure how to use it for 1.8.

**Comment by nicolargo** [comment]
Please use nicolargo/glances:alpine-dev image, it should solve the issue (the current stable/last version did not include all the deps).

It should be solved when the Glances 3.3.0 image will be release.
